src/main/webapp/WEB-INF/tags/instructor/search/feedbackResponse.tag

```diff
@@ -27,12 +27,10 @@
 <tr>
   <td>
     <ul class="list-group comments" id="responseCommentTable-${fsIndx}-${qnIndx}-${responseIndex}"
-        <c:if test="${empty responseRow.feedbackResponseComments}">style="display:none"</c:if>>
-
-      <c:forEach items="${responseRow.feedbackResponseComments}" var="frc" varStatus="i">
-        <shared:feedbackResponseCommentRow frc="${frc}" firstIndex="${fsIndx}" secondIndex="${qnIndx}" thirdIndex="${responseIndex}" frcIndex="${i.count}" />
-      </c:forEach>
-
+      <c:if test="${empty responseRow.feedbackResponseComments}">style="display:none"</c:if>>
```

Comment:
This should be indented an additional 2 spaces (this is a continuation of the `ul` tag, not an inner element).